### PR TITLE
Feat/api

### PR DESCRIPTION
**Pull request by nmrichards**
Added amplify api.

### CODE REVIEW

**Comment by wallace41290** [comment]
Screen Shot 2024-01-18 at 7 58 31 PM
You've got a failure in the pipeline, when it just spits out files, I think that's usually the formatted complaining. You should be able to locally run `npm run pr-check` to produce the same results and fix issues.
1. the nested vscode settings can be deleted
2. the graphql schema either needs to be formatted or put in the prettierignore

`npm run format` will try to automatically format files

File: apps/trucking-kit/.vscode/settings.json
Comment (wallace41290) [inline apps/trucking-kit/.vscode/settings.json:1]: delete this file

File: apps/trucking-kit/.eslintignore
Comment (wallace41290) [inline apps/trucking-kit/.eslintignore:1]: not sure what this is. do we need it? if you delete it and then run the linter, does it still pass?

File: AWSCLIV2.pkg
Comment (wallace41290) [inline AWSCLIV2.pkg:1]: I don't think we need this in git, can you delete?

File: libs/trucking-kit/companies/companies.component.ts
Comment (wallace41290) [inline libs/trucking-kit/companies/companies.component.ts:1]: For now, can you just move these files to:
`libs/trucking-kit/home/feature-home/src/lib/companies/companies.component.{ts,html,css}`

and keep it in the same feature-home lib for now?

**Review by wallace41290 — CHANGES_REQUESTED** [review]

File: apps/trucking-kit/amplify/backend/api/truckingkit/schema.graphql
Comment (wallace41290) [inline apps/trucking-kit/amplify/backend/api/truckingkit/schema.graphql:1]: Did we decide to leave the user for now? Let's try to keep this PR as slim as possible

Comment (nmrichards) [reply apps/trucking-kit/amplify/backend/api/truckingkit/schema.graphql:1]: With this, if we don't have a user, how do ensure the Company/user relationship? Are you able to establish the relationship within Cognito alone? Either by storing the company information within the user or user information within the company (or maybe both)?

Comment (nmrichards) [reply apps/trucking-kit/amplify/backend/api/truckingkit/schema.graphql:1]: I'm looking into this as I really don't know. Maybe they do some backend magic to link those two together easier?

**Review by wallace41290 — DISMISSED** [review]
LGTM

**Pull request merged**